tests/app/utilities/test_schema.py

```diff
@@ -44,3 +45,12 @@ def test_transform_schema_name_bad_case_type_raises_error():
 )
 def test_get_allowed_languages(schema_name, launch_language, expected):
     assert get_allowed_languages(schema_name, launch_language) == expected
+
+
+def test_get_schema_path_map():
+    schema_path_map = get_schema_path_map()
+
+    assert "en" in schema_path_map.keys()
+    assert "cy" in schema_path_map.keys()
+    assert "ga" in schema_path_map.keys()
+    assert "eo" in schema_path_map.keys()
```

Comment:
Some more tests here would make it clearer what structure is generated from this method.